alpine/Dockerfile

```diff
@@ -0,0 +1,27 @@
+ARG ARCH=""
+FROM        ${ARCH}alpine:latest as alpine
+
+RUN apk upgrade -U && \
+    apk add tzdata ca-certificates
+
+# Use the busybox.static to avoid dynamic library dependencies.
+RUN apk add busybox-static && \
+    mv /bin/busybox.static /bin/busybox && \
+    /bin/busybox --install -s /bin
+
+# remove extra package
+RUN apk del libc-utils
+
+# remove apk files and directories
+RUN apk del apk-tools && \
+    find / -name apk -prune -exec rm -rf {} ";"
+
+
+# remove extraneous folders
+RUN rm -rf media mnt opt run srv
+
+FROM        scratch
+MAINTAINER  The Prometheus Authors <prometheus-developers@googlegroups.com>
+
+COPY --from=alpine / /
```

Comment:
This is copying everything, so, doing `FROM scratch` seems unnecessary. Our other images only copy over specific configuration files in order to make sure busybox is the only binary in the final image.

Reply:
I'm not sure what you're asking for here.

Reply:
I think what @SuperQ is suggesting is instead of removing the unneeded things in the base image and then copying everything here, we should use explicitly only COPY here what we need.
Can you only copy busybox here and then run `/bin/busybox --install -s /bin` in the final image?